app/scripts/controllers/detect-tokens.js

```diff
@@ -208,8 +218,8 @@ export default class DetectTokensController {
     this.interval = DEFAULT_INTERVAL;
   }
 
-  getChainIdFromNetworkStore(network) {
-    return network?.store.getState().providerConfig.chainId;
+  getChainIdFromNetworkStore() {
```

Comment:
The argument for this method was removed because the thing it is trying to access is already accessible as a property.